demo/src/test/java/com/mongodb/jdbc/utils/TestUtils.java

```diff
@@ -0,0 +1,439 @@
+package com.mongodb.jdbc.utils;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public abstract class TestUtils {
+
+    protected Connection m_conn;
+    protected DatabaseMetaData m_dbMeta;
+
+    protected abstract Connection connect() throws Exception;
+
+    @BeforeEach
+    private void execConnection() throws Exception {
+        m_conn = connect();
+        m_dbMeta = m_conn.getMetaData();
+    }
+
+    @AfterEach
+    private void execDeconnection() throws Exception {
+        m_conn.close();
+    }
+
+    @Test
+    void testConnection() throws Exception {
+
+        System.out.println(m_dbMeta.getDriverName());
+        System.out.println(m_dbMeta.getDriverVersion());
+
+        System.out.println("Connected !");
+    }
+
+    @Test
+    void testExecute() throws Exception
+    {
+        runExecuteQuery("select * from bar", true);
+    }
+
+    @Test
+    void testExecutePrintMetadata() throws Exception
+    {
+        runExecuteQuery("select * from bar", false);
+    }
+
+    @Test
+    void testGetCatalogs() throws Exception
+    {
+        ResultSet rs = m_dbMeta.getCatalogs();
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetTables() throws Exception
+    {
+        //ResultSet rs = m_dbMeta.getTables(null, null, null, null);
```

Comment:
What's the purpose of this comment and the ones below it?

Reply:
Ah, I see. Can you add an explanatory comment for readability?

Reply:
I also decided to rename the helpers from testXX to tryXX in hope that it differentiates them from actual unit tests.